packages/locale-loader/lib/isLocaleFile/index.js

```diff
@@ -7,10 +7,22 @@ const fileRegExp = /\.(js)$/i;
  * @param {String} filename
  * @returns {Boolean}
  */
-export default function (filename) {
+export default function isLocaleFile(filename) {
   if (!fileRegExp.test(filename)) {
     return false;
   }
   const name = filename.replace(fileRegExp, '');
   return localeFileRegExp.test(name);
 }
+
+function localeFileFilter(locales = ['en-US']) {
+  return (typename) => {
+    if (locales.includes(typename)) return true;
```

Comment:
Use ramda for complex array manipulation please. Thins like array.find, array.findIndex, array.includes, are all potentially troublesome. I know that this code is targeting node, and should include those functionalities. But I think it's a better practice to use util lib like ramda to cover these cases.

Quite frankly I am not sure if this function is even necessary.
You can easily do ```array.filter(item => locales.includes(item))``` in the place you need this filter creator. Personally I think this is unnecessary as this filter is too simple.